docs/references/subcommands/container.md

```diff
@@ -33,6 +33,17 @@ fossa container analyze debian
 # Explicit remote image via docker.your-org.com
 fossa container analyze docker.your-org.com/project/image
 ```
+You can provide, `--only-system-deps` to only analyze dependencies originating from following package managers:
+
+- dpkg
+- rpm
+- alpine
+
+> Performing `fossa container analyze <IMAGE> --only-system-deps` will match scanner behaviour that of
+> previous FOSSA container scanner (same as all CLI prior to v3.5.0)
```

Comment:
This sounds a little clunky, maybe something like this. I think its the "that of" part throwing me off.

```suggestion
> Performing `fossa container analyze <IMAGE> --only-system-deps` will match the behavior of
> the previous FOSSA CLI container scanner (same as all CLI prior to v3.5.0)
```